cmd/cluster/core/create.go

```diff
@@ -68,9 +68,9 @@ func bindCoreOptions(opts *RawCreateOptions, flags *pflag.FlagSet) {
 	flags.StringVar(&opts.ReleaseImage, "release-image", opts.ReleaseImage, "The OCP release image for the cluster")
 	flags.StringVar(&opts.PullSecretFile, "pull-secret", opts.PullSecretFile, "File path to a pull secret.")
 	flags.StringVar(&opts.ControlPlaneAvailabilityPolicy, "control-plane-availability-policy", opts.ControlPlaneAvailabilityPolicy, "Availability policy for hosted cluster components. Supported options: SingleReplica, HighlyAvailable")
-	flags.BoolVar(&opts.Render, "render", opts.Render, "Render output as YAML to stdout instead of applying")
+	flags.BoolVar(&opts.Render, "render", opts.Render, "Render output as YAML to stdout instead of applying, also use --render-sensitive to render sensitive information")
```

Comment:
maybe need to clarify what `sensitive information` is
```suggestion
	flags.BoolVar(&opts.Render, "render", opts.Render, "Render output as YAML to stdout instead of applying. Note: secrets, foo, bar, etc are not rendered by default, use --render-sensitive to render sensitive information")
```